### ISSUE TITLE
Design Intent - Accessibility Feedback - EDM and VEBT team, GI Bill Comparison Tool homepage

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#99573]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [x] **Should:** There's not guidance in the design system for this and there are places already in production on VA.gov where this "rule" is already being broken, but there's some evidence from user research that headings that are also links can be confusing to screen reader users. Basically, interactions become a lot easier when an element is one thing or another thing, but not trying to serve multiple roles. I would encourage you to explore an option where the headings are just headings, but an action link or similar appears beneath the content for that section.
- [x] **Consider:** We often (but again, not exclusively or consistently) try to stay away from multi-column page layouts, as screen magnification users can sometimes miss anything that's floating too far off to the right. I think what's here is probably fine (columns aren't far apart, similar amounts of content in each column), but consider exploring other layouts to see if there's anything that sticks. Definitely not a must, but an opportunity to potentially remove friction for some users.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by cassidy-beach** [comment]
Should: There's not guidance in the design system for this and there are places already in production on VA.gov where this "rule" is already being broken, but there's some evidence from user research that headings that are also links can be confusing to screen reader users. 
Further discussion in [Design Intent - Design Feedback](https://github.com/department-of-veterans-affairs/va.gov-team/issues/99543#event-15733018527) comments 
Created [VEBT-1368](https://jira.devops.va.gov/browse/VEBT-1368) to address this in the future

**Comment by cassidy-beach** [comment]
[VEBT-1368](https://jira.devops.va.gov/browse/VEBT-1368) to be taken on at a later date, addresses this issue
